packages/@okta/vuepress-site/docs/concepts/policies/index.md

```diff
@@ -98,6 +99,21 @@ You can create an authentication policy specifically for the app or create a few
 
 Use the [Authentication Policies page](https://help.okta.com/okta_help.htm?type=oie&id=ext-create-auth-policy) to modify an app's sign-on policy or switch to a different policy. See [Configure a global session policy and an authentication policy](/docs/guides/configure-signon-policy/main/). Also, you can [merge duplicate authentication policies with identical rules](https://help.okta.com/okta_help.htm?type=oie&id=ext-merge-auth-policies) to simplify policy management.
 
+##### Okta account management policy
+
+<ApiLifecycle access="ie" />
+<ApiLifecycle access="ea" />
+
+The Okta account management policy is a type of authentication policy. It defines requirements when users enroll in authenticators, recover their passwords, and unlock their accounts. The policy uses a rule-based framework to enforce phishing resistance throughout the user journey, from onboarding to authentication and recovery.
```

Comment:
```suggestion
The Okta account management policy is a type of authentication policy. It defines requirements when users enroll in authenticators, recover their passwords, and unlock their accounts. Its rule-based framework lets you enforce phishing resistance throughout the user journey, from onboarding to authentication and recovery.
```

Reply:
this change to the third sentence will make it match my intro, where I got some feedback that the framework itself didn't enforce PR, but it enabled the option to enforce it.